linux_os/guide/system/bootloader-grub2/grub2_password/oval/shared.xml

```diff
@@ -31,12 +25,12 @@
     <unix:filepath>/boot/grub2/grub.cfg</unix:filepath>
   </unix:file_object>
 
-  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="superuser is defined in /boot/grub2/grub.cfg files. Superuser is not root, admin, or administrator" id="test_bootloader_superuser" version="1">
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="superuser is defined in /boot/grub2/grub.cfg files." id="test_bootloader_superuser" version="2">
     <ind:object object_ref="object_bootloader_superuser" />
   </ind:textfilecontent54_test>
-  <ind:textfilecontent54_object id="object_bootloader_superuser" version="1">
+  <ind:textfilecontent54_object id="object_bootloader_superuser" version="2">
     <ind:filepath>/boot/grub2/grub.cfg</ind:filepath>
-    <ind:pattern operation="pattern match">^[\s]*set[\s]+superusers=\"(?i)(?!root|admin|administrator)(?-i).*\"$</ind:pattern>
+    <ind:pattern operation="pattern match">^[\s]*set[\s]+superusers=\".*\"$</ind:pattern>
```

Comment:
the regex allows username `""` or invalid user names

Reply:
I think that this is fine - we are supposed to check whether the username doesn't violate a requirement, and it's grub's responsibility to handle nuances s.a. invalid or empty usernames.

Reply:
Good point.